src/Compilers/CSharp/Portable/BoundTree/Constructors.cs

```diff
@@ -417,6 +417,11 @@ internal sealed partial class BoundBinaryOperator
             var uncommonData = UncommonData.CreateIfNeeded(constantValueOpt, methodOpt, constrainedToTypeOpt, OriginalUserDefinedOperatorsOpt);
             return Update(operatorKind, uncommonData, resultKind, left, right, type);
         }
+
+        public BoundBinaryOperator Update(UncommonData uncommonData)
```

Comment:
The only use of the overload is to add `interpolatedStringHandlerData` to a `BoundBinaryOperator`. Consider adding `Update(InterpolatedStringHandlerData)` instead.